### ISSUE TITLE
Feedback for “Image”

### ISSUE THREAD

**Issue by brogr** [samvera-labs/clover-iiif#234]
Currently, the documentation gives this example for Next.js:

```
import dynamic from "next/dynamic";
 
const Image = dynamic(
  () => import("@samvera/clover-iiif").then((Clover) => Clover.Image),
  {
    ssr: false,
  },
);
```

That doesn't work. Correct would be:

```
const Image = dynamic(
  () => import("@samvera/clover-iiif/image").then((Image) => Image),
  {
    ssr: false,
  },
);
```

**Comment by brogr** [comment]
Hey @mathewjordan, as people keep [running into this issue](https://github.com/samvera-labs/clover-iiif/issues/242#issuecomment-2627335030), would you mind updating the docs?

**Comment by mathewjordan** [comment]
Certainly @brogr. What is the correct implementation? Yours or the one mentioned in #242?

**Comment by brogr** [comment]
@mathewjordan, yep, that’s what I’ve been using lately too. It corresponds to what nextjs is recommending for non-named exports: https://nextjs.org/docs/app/guides/lazy-loading#skipping-ssr

Works for the Viewer too
